### ISSUE TITLE
[Bug] 2.3 pre release Not working Reshade

### ISSUE THREAD

**Issue by zerr10** [Lyall/MGSHDFix#117]
if i put this files on folder reshade doesn't want to work how fix this ?? i want resolution full screen.

reshade work if i no put this files on folder but then i no have full screen cant u fix this ??

**Comment by zerr10** [comment]
there is no other way?? to make the resolution full screen? I don't think about aspect ratio in reshade because it causes errors in displaying graphics, e.g. stripes appear on the water

**Comment by ShizCalev** [comment]
Please post both your MGSHDFix & Reshade log files.

**Comment by EV3RGR33N** [comment]
Hey there, I just wanted to add that I experienced this myself. With ReShade (6.3.3, using dxgi.dll) and MGSHDFix 2.3 Pre Release, the game would simply CTD. Also, I did remove the d3d11.dll from MGSHDFix 2.2. 
[MGS2_2_3_PreRelease_Reshade6_3_3.zip](https://github.com/user-attachments/files/17632998/MGS2_2_3_PreRelease_Reshade6_3_3.zip)

I tried messing with the settings offered in MGSHDFix.ini. I found out that the problem is seemingly not the game itself, it's the launcher. Setting SkipLauncher to True does the trick though - boots straight to the game itself, and Reshade works a-ok. 

**TLDR Here is a workaround:
Open MGSHDFix.ini and set "SkipLauncher" to "True". This will skip the launcher, and ReShade will work properly. Returning back to the Main Menu from the game will crash.
If you want to use the resolution scaling features from MGSHDFix, make sure to launch the game without Reshade at least once, and set the graphics options to "Default/Original.".**

**Comment by ShizCalev** [comment]
to be more specific with what the issue is here - reshade causes the launchers to crash. 

it's entirely an issue between reshade and the unity launcher, not an issue with mgshdfix.

**Comment by ShizCalev** [comment]
Closing, as this is a bug with reshade / occurs even without MGSHDFix.
